argocd/loki-stack/Chart.yaml

```diff
@@ -2,6 +2,9 @@
 apiVersion: "v2"
 name: "loki-stack"
 version: "0"
+description: |
+  This chart installs loki-stack. 
+  loki is the log aggregation system inspired by Prometheus.
```

Comment:
```suggestion
  Loki is a log aggregation system made by Grafana Labs, and inspired by Prometheus.
```

Reply:
Add details on how this sets up automatic log gathering and piping into Grafana to create dashboards.